doc/sphinx/locale/es/LC_MESSAGES/notifications.po

```diff
@@ -0,0 +1,212 @@
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.5.1\n"
+
+#: ../../notifications/index.rst:5
+msgid "Notification Templates"
+msgstr "Plantillas de Notificationes"
+
+#: ../../notifications/index.rst:7
+msgid "Brand administators can configure the notifications sent by IvozProvider:"
+msgstr ""
+"Los administradores de marca pueden configurar las notificaciones "
+"enviadas por IvozProvider:"
+
+#: ../../notifications/index.rst:9
+msgid "Email sent when a new voicemail is received"
+msgstr "Correos electrónicos enviados cuando se recibe un buzón de voz nuevo."
+
+#: ../../notifications/index.rst:11
+msgid "Email sent when a new fax is received"
+msgstr "Correos electrónicos enviados cuando se recibe un fax nuevo."
+
+#: ../../notifications/index.rst:13
+msgid "When no custom notification is configured, default ones will be used"
+msgstr "Si no se configuran notificaciones, se emplearán notificaciones genéricas"
+
+#: ../../notifications/index.rst:15
+msgid ""
+"Notifications are created in two steps: Create a notification type and "
+"add contents to the notification for each required language."
+msgstr ""
+"Las plantillas de notificaciones son creadas en dos pasos: Crear una tipo"
+" de notificación y añadir contenido a la notificación para cada lenguaje "
+"necesario."
+
+#: ../../notifications/index.rst:20
+msgid "Creating a new notification"
+msgstr "Crear una nueva notificación"
+
+#: ../../notifications/index.rst:22
+msgid ""
+"Brand administrators can create new notification templates in **Brand "
+"configuration** > **Notification templates**:"
+msgstr ""
+"Los administradores de marca puede crear nuevas plantillas de "
+"notificacion en **Configuración de marca** > **Plantillas de "
+"notificación**"
+
+#: ../../notifications/index.rst:24
+msgid "Fields are nearly self-explanatory:"
+msgstr "Los campos son auto-explicativos"
+
+#: ../../notifications/index.rst:27
+msgid "Name"
+msgstr "Nombre"
+
+#: ../../notifications/index.rst:29
+msgid "Used to identificate this notification template"
+msgstr "Empleado para identificar esta plantilla de notificación"
+
+#: ../../notifications/index.rst:30
+msgid "Type"
+msgstr "Tipo"
+
+#: ../../notifications/index.rst:32
+msgid ""
+"Determine the notification type. Each notification type has its own "
+"substitution variables avaiable to replace the contents of the subject "
+"and body."
+msgstr ""
+"Determina el tipo de notificación. Cada tipo de notificación tiene sus "
+"propias variables de sustitución disponibles para poder replazar en los "
+"contenidos del asunto y cuerpo."
+
+#: ../../notifications/index.rst:37
+msgid "Adding Notification contents"
+msgstr "Añadiendo contenidos a la notificación"
+
+#: ../../notifications/index.rst:39
+msgid ""
+"Once the notification has been created, you can add different language "
+"contents. IvozProvider will automatically use the correct language based "
```

Comment:
proper sound better than correct to me, what do you think about it?

Reply:
Replaced.